app/hooks/useLanguage.tsx

```diff
@@ -1,39 +1,33 @@
-import type { Dispatch, FunctionComponent, PropsWithChildren, SetStateAction } from 'react'
 import { createContext, useContext, useEffect, useState } from 'react'
+import type { Dispatch, FunctionComponent, PropsWithChildren, SetStateAction } from 'react'
 
-export enum Language {
-  EN = 'en-US',
-  DE = 'de-DE'
-}
+const languages = ['en', 'de'] as const
```

Comment:
Why do we get rid of the enum?

Reply:
basically boils down to enums being an anti-pattern in typescript (somewhat)